src/components/CurrencyInputPanel/FiatValue.tsx

```diff
@@ -29,17 +39,33 @@ export function FiatValue({
   const p = Number(fiatValue?.toFixed())
   const visibleDecimalPlaces = p < 1.05 ? 4 : 2
 
+  useEffect(() => {
+    if (isLoading) {
+      const timeout = setTimeout(() => setShowLoadingPlaceholder(true), 200)
+      return () => clearTimeout(timeout)
+    } else {
+      setShowLoadingPlaceholder(false)
+      return
+    }
+  }, [isLoading])
+
   return (
     <ThemedText.DeprecatedBody fontSize={14} color={theme.textSecondary}>
-      {fiatValue && <>${fiatValue?.toFixed(visibleDecimalPlaces, { groupSeparator: ',' })}</>}
-      {priceImpact ? (
-        <span style={{ color: priceImpactColor }}>
-          {' '}
-          <MouseoverTooltip text={t`The estimated difference between the USD values of input and output amounts.`}>
-            (<Trans>{priceImpact.multiply(-1).toSignificant(3)}%</Trans>)
-          </MouseoverTooltip>
-        </span>
-      ) : null}
+      {showLoadingPlaceholder ? (
+        <FiatLoadingBubble />
+      ) : (
+        <div>
+          {fiatValue && <>${fiatValue?.toFixed(visibleDecimalPlaces, { groupSeparator: ',' })}</>}
```

Comment:
might be a good opportunity to use @uniswap/conedison

Reply:
good idea done